en/community/conferences/index.md

```diff
@@ -9,7 +9,7 @@ conferences, where they get together to share reports on
 work-in-progress, discuss the future of Ruby, and welcome newcomers to
 the Ruby community.
 
-[RubyConferences.org][rc] is a simple list of Ruby-specific conferences,
+[https://www.rubyevents.org/][rc] is a simple list of Ruby-specific conferences,
```

Comment:
The title of site says "RubyEvents.org", so `[RubyEvents.org](https://www.rubyevents.org/)` would be better.

```suggestion
[RubyEvents.org][rc] is a simple list of Ruby-specific conferences,
```